### PR TITLE
feat(plugin): CatPictures

### PR DESCRIPTION
**Pull request by Jannniss**
This plugin adds a command to get a cat picture from cataas api cause everyone likes cats.

### CODE REVIEW

**Comment by cheesesamwich** [comment]
add yourself to the devs construct if you're PRing a plugin

File: src/plugins/catPictures/index.tsx [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+                if (response != null) {
+                    let file = new File([response], "cat.jpeg", { type: "image/jpeg" })
+                    setTimeout(() => UploadHandler.promptToUpload([file], ctx.channel, draft_type), 10);
+                }
+            }
+        },
+    ]
+});
+
+function returnFloatWithParameter(number, parameter) {
```
Comment (MrDiamondDog) [inline src/plugins/catPictures/index.tsx:133]: This function should be named better (i can tell from the strings in this function that this should have a different name)

Also add types to the parameters, we're not in js

File: src/plugins/catPictures/index.tsx [hunk 2]
```diff
@@ -0,0 +1,239 @@
+/*
```
Comment (MrDiamondDog) [inline src/plugins/catPictures/index.tsx:1]: Use new header

File: src/plugins/catPictures/index.tsx [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+                        name: "negate"
+                    },
+                    {
+                        label: "custom",
+                        value: "custom",
+                        name: "custom (use custom options)"
+                    }]
+                },
+                {
+                    name: "saturation-multiplier",
```
Comment (MrDiamondDog) [inline src/plugins/catPictures/index.tsx:76]: Realistically, no one is going to use some of these options. Consider removing some of them

Comment (Jannniss) [reply src/plugins/catPictures/index.tsx:64]: removed color options and second brightness option

**Comment by Vendicated** [comment]
hi, thank you a lot for your contribution!

this doesn't seem very useful and could easily be replaced with a simple discord bot (or better yet a [user installable app](https://discord.com/developers/docs/tutorials/developing-a-user-installable-app))

as such i'll be rejecting this, sorry

**Pull request closed without merging**